rudder-core/src/main/scala/com/normation/rudder/services/system/DatabaseManager.scala

```diff
@@ -103,17 +103,21 @@ class DatabaseManagerImpl(
    }
 
    def archiveEntries(date : DateTime) =  {
-     reportsRepository.archiveEntries(date)
+     val archiveReports = reportsRepository.archiveEntries(date)?~! "An error occured while archiving reports"
+     val archiveNodeConfigs = expectedReportsRepo.archiveNodeConfigurations(date) ?~! "An error occured while archiving Node Configurations"
+
+      // Accumulate errors, them sum values
+     (Control.bestEffort(Seq(archiveReports, archiveNodeConfigs)) (identity)).map(_.sum)
    }
 
 
    def deleteEntries(date : DateTime) : Box[Int] = {
      val reports = reportsRepository.deleteEntries(date) ?~! "An error occured while deleting reports"
      val nodeConfigs = expectedReportsRepo.deleteNodeConfigIdInfo(date) ?~! "An error occured while deleting reports"
```

Comment:
This function makes request on expectedreportsnodes which was removed too I guess it can be removed too